packages/tradershub/src/components/Base/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type TProps = {
+    alignment?: 'bottom' | 'left' | 'right' | 'top';
+    className?: string;
+    isVisible: boolean;
+    message: string;
+};
+
+const Tooltip: React.FC<React.PropsWithChildren<TProps>> = ({
+    alignment = 'left',
+    children,
+    className,
+    isVisible,
+    message,
+}) => {
+    return (
+        <div className={`relative w-max h-max ${className ?? ''}`}>
```

Comment:
can use qtmerge for this like button group

Reply:
```suggestion
        <div className={qtMerge('relative w-max h-max', className)}>
```